app/angular/index.d.ts

```diff
@@ -29,4 +30,5 @@ declare module '@storybook/angular' {
   export function addDecorator(decorator: any): IApi;
   export function configure(loaders: () => NodeRequire, module: NodeModule): void;
   export function getStorybook(): IStoribookSection[];
+  export function clearDecorators(): void;
```

Comment:
Adding this here makes it a public API. I think we don't want to expose this method.